voucher/utils.py

```diff
@@ -57,27 +57,35 @@ def get_eligible_coupon_choices(voucher):
     course_matches = None
     # Search for an exact match if all inputs exist
     if voucher.course_id_input and voucher.course_title_input:
-        course_matches = CourseRun.objects.filter(
-            course__readable_id__exact=voucher.course_id_input,
-            course__title__exact=voucher.course_title_input,
-            start_date__date=voucher.course_start_date_input,
+        course_matches = (
+            CourseRun.objects.filter(
+                course__readable_id__exact=voucher.course_id_input,
+                course__title__exact=voucher.course_title_input,
+                start_date__date=voucher.course_start_date_input,
```

Comment:
Could just add `live=True` here instead of adding a new `CourseRunQuerySet` and `CourseRunQueryManager` but either way works, your choice.

Reply:
we have those mangers for Programs and Courses so I thought moving forward we can have same filters for course runs as well.